### PR TITLE
Feature translation

### PR DESCRIPTION
**Pull request by peyman-mohtashami**

### CODE REVIEW

File: src/app/pages/auth/services/keycloak.auth.service.ts [hunk 1]
```diff
@@ -238,6 +244,7 @@ export class KeycloakAuthService extends AuthService {
     const responseMode = 'query'
     const responseType = 'code'
     const scope = 'openid'
+    this.language = this.localization.getLanguage()
```
Comment (nivemaham) [inline src/app/pages/auth/services/keycloak.auth.service.ts:188]: Isn't this an asynchronous call? If it is, maybe adding the return and then would be better

Comment (peyman-mohtashami) [reply src/app/pages/auth/services/keycloak.auth.service.ts:188]: No, it is not async call.
```localization.service.ts```
```js
getLanguage(): LanguageSetting {
    return this.language
  }
```

File: config.xml [hunk 2]
```diff
@@ -138,6 +137,5 @@
     </plugin>
     <plugin name="cordova-plugin-inappbrowser" spec="^3.2.0" />
     <plugin name="cordova-plugin-camera" spec="^4.0.3" />
-    <engine name="ios" spec="~5.0.1" />
-    <engine name="android" spec="^8.1.0" />
+    <engine name="ios" spec="^5.0.1" />
```
Comment (nivemaham) [inline config.xml:141]: Please add the android back to supported engines. Not sure how you were able to build the app for android without this

Comment (peyman-mohtashami) [reply config.xml:141]: I didn't change these lines manually. But I can generate app-debug and also build app-release apks.
BTW Maybe it causes some problems in the future tests, so I'll add ```<engine name="android" spec="^8.1.0" />```

**Review by nivemaham — APPROVED** [review]
LGTM

**Pull request merged**